lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+    attr_reader :list, :choices
+    def initialize 
+        @list = [
+            {:dish => 'green curry', :price => 5.99},
+            {:dish => 'panang', :price => 7.5},
+            {:dish => 'veg tempura', :price => 4.5},
+            {:dish => 'rice', :price => 2.5}
+            ]
+        @choices = []
+    end
+
+    def select(dish_string, quantity)
+        @list.each do |details|
+            if details[:dish] == dish_string
+                details.merge!({:quantity => quantity})
+                return @choices.push(details)
+            end
+        end
+        raise 'dish not found'
+    end
+end
```

Comment:
Did you run Rubocop? There are a few minor formatting issues such as not having the newlines at the end of your files to ensure it's compliant with the usual Ruby approach.